openhands/server/utils.py

```diff
@@ -18,6 +19,15 @@ async def get_conversation_store(request: Request) -> ConversationStore | None:
     return conversation_store
 
 
+async def generate_unique_conversation_id(
```

Comment:
NIT: I think we have duplicate logic in `create_new_conversation` function; we should use this helper instead